integration/ros2_workspace/src/test_py_module/setup.py

```diff
@@ -10,10 +10,8 @@
     packages=setuptools.find_packages(),
     zip_safe=True,
     install_requires=[
-        'annoy<=1.8.5',
```

Comment:
I was the one who added these back when dashing support was added. I do not remember why I decided to version lock them. The tests are passing for both dashing and foxy, so I think we can proceed.